core/templates/pages/exploration-editor-page/statistics-tab/statistics-tab.component.ts

```diff
@@ -38,15 +41,17 @@ angular.module('oppia').component('statisticsTab', {
   controller: [
     '$q', '$scope', '$uibModal', 'AlertsService', 'ComputeGraphService',
     'ExplorationDataService', 'ExplorationStatsService',
-    'ReadOnlyExplorationBackendApiService', 'StateInteractionStatsService',
-    'StatesObjectFactory', 'UrlInterpolationService',
+    'ReadOnlyExplorationBackendApiService', 'RouterService',
+    'StateInteractionStatsService', 'StatesObjectFactory',
+    'UrlInterpolationService',
     function(
         $q, $scope, $uibModal, AlertsService, ComputeGraphService,
         ExplorationDataService, ExplorationStatsService,
-        ReadOnlyExplorationBackendApiService, StateInteractionStatsService,
-        StatesObjectFactory, UrlInterpolationService) {
+        ReadOnlyExplorationBackendApiService, RouterService,
+        StateInteractionStatsService, StatesObjectFactory,
+        UrlInterpolationService) {
       const expId = ExplorationDataService.explorationId;
-
+      this.directiveSubscriptions = new Subscription();
```

Comment:
Move this to the very top of the function so as to not break up the logical groupings.

Reply:
Done